wgs/paired-fastq-to-unmapped-bam/README.md

```diff
@@ -0,0 +1,7 @@
+The workflow in this repository is the Paired FastQ to Unmapped BAM workflow provided by the Broad Institute.
+
+A copy of the paired-fastq-to-unmapped-bam.wdl file was made on February 23, 2018 from the Broad's github repo:
```

Comment:
A copy of the February 23, 2018 version of the `paired-fastq-to-unmapped-bam.wdl` file was made from the Broad Institute's github repo:

Reply:
This change has been made